include/lmms_basics.h

```diff
@@ -97,13 +97,13 @@ inline float typeInfo<float>::minEps()
 }
 
 template<>
-inline bool typeInfo<float>::isEqual( float _x, float _y )
+inline bool typeInfo<float>::isEqual( float x, float y )
 {
-	if( likely( _x == _y ) )
+	if( unlikely( x == y ) )
```

Comment:
Don't these two functions have opposite meaning?  Shouldn't the return value be inverted if changed from `likely()` to `unlikely()` ?